src/main/java/core/mate/academy/model/TruckProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer {
```

Comment:
You are using a raw type for MachineProducer. As per the checklist, you should parametrize the MachineProducer interface. For example, it should be `public class TruckProducer implements MachineProducer<Truck>`.